src/mail/mjml/partial/footer.mjml

```diff
@@ -1,8 +1,7 @@
-<mj-include path="./course_greetings.mjml" />
-
-<mj-section background-color="#000000" padding="15px">
-  <mj-column vertical-align="top" width="100%">
-    <mj-text align="center" color="#FFFFFF" font-size="12px" padding="5px 25px">{%trans "This mail has been sent to" %} {{email}} {%trans "by Joanie"%}
+<mj-section padding="0">
+  <mj-column>
+    <mj-text mj-class="text--small" align="center" padding="20px 40px">
+      {%trans "This mail has been sent to" %} <a href="#">{{email}}</a> {%trans "by Joanie"%}
```

Comment:
I'm not sure we want to mention joanie to the enduser. Should be "France Université Numérique" in our case, so something configurable?
```suggestion
      {%blocktrans with email=email %}This mail has been sent to <a href="#">{{email}}</a> by Joanie"{% endblocktrans %}
```

Reply:
Good idea !